### PR TITLE
RAB-796: Display proper error message when cannot open file

### PR DESCRIPTION
**Pull request by ValentinMumble**
image

### CODE REVIEW

**Review by ghoussard — APPROVED** [review]

File: src/Akeneo/Platform/Bundle/ImportExportBundle/Infrastructure/Validation/FilePath.php [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+declare(strict_types=1);
+
 namespace Akeneo\Platform\Bundle\ImportExportBundle\Infrastructure\Validation;
 
 use Symfony\Component\Validator\Constraint;
 
 class FilePath extends Constraint
 {
-    public const UNSUPPORTED_EXTENSION = 'akeneo.job_automation.validation.file_path.unsupported_extension';
+    public const UNSUPPORTED_EXTENSION = 'akeneo.automation.validation.file_path.unsupported_extension';
```
[the remark this answers is not in the document]
Comment (ValentinMumble) [reply src/Akeneo/Platform/Bundle/ImportExportBundle/Infrastructure/Validation/FilePath.php:11]: That's how they are defined https://github.com/akeneo/pim-enterprise-dev/blob/master/components/job-automation/back/src/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml#L1

[the remark this answers is not in the document]
Comment (ghoussard) [reply src/Akeneo/Platform/Bundle/ImportExportBundle/Infrastructure/Validation/FilePath.php:11]: Ah!

[the remark this answers is not in the document]
Comment (ghoussard) [reply src/Akeneo/Platform/Bundle/ImportExportBundle/Infrastructure/Validation/FilePath.php:11]: So I think, we have to fix the translation file, job_automation is more accurate than automation

[the remark this answers is not in the document]
Comment (ValentinMumble) [reply src/Akeneo/Platform/Bundle/ImportExportBundle/Infrastructure/Validation/FilePath.php]: I fixed it

**Pull request merged**